src/components/lyne-teaser-thumbnail/styles/lyne-teaser-thumbnail.component.scss

```diff
@@ -0,0 +1,94 @@
+@mixin image-aspect-ratio-4-3 {
+  aspect-ratio: 4 / 3;
+}
+
+/* ************************************************* */
+/* Layout & spacing                                  */
+/* ************************************************* */
+.teaser-thumbnail {
+  border: none;
+  text-decoration: none;
+  text-align: start;
+  display: inline-block;
+
+  &:focus,
+  &:hover {
+    .teaser-thumbnail__image {
+      @include shadow-level-9-hard;
+      transform: scale(1.02);
+      transition: var(--animation-duration-4x) var(--animation-easing);
+    }
+  }
+}
+
+.teaser-thumbnail__image {
+  @include image-aspect-ratio-4-3;
+
+  border-radius: pxToRem(var(--border-radius-4x));
+  margin-bottom: pxToRem(var(--spacing-fixed-3x));
+  width: auto;
+  height: auto;
+}
+
+.teaser-thumbnail__content {
+  display: flex;
+}
+
+.teaser-thumbnail__inner { 
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+// Layout switch
+.teaser-thumbnail {
+  &--isStacked {
+    .teaser-thumbnail__inner {
+      flex-direction: column;
+    }
+    .teaser-thumbnail__image {
+      max-width: 280px;
```

Comment:
Better don't use px values (rem suggested).
Also please add a variable in your :host {} to avoid magic numbers.